cmd/ascii2der/scanner.go

```diff
@@ -284,6 +284,51 @@ again:
 		if s.pos.Offset+1 < len(s.text) && s.text[s.pos.Offset+1] == '"' {
 			return s.parseUTF32String()
 		}
+	case 'b':
+		if s.pos.Offset+1 < len(s.text) && s.text[s.pos.Offset+1] == '`' {
+			s.advance() // Skip the b.
+			s.advance() // Skip the `.
+			bitStr, ok := s.consumeUpTo('`')
+			if !ok {
+				return token{}, &parseError{s.pos, errors.New("unmatched `")}
+			}
+
+			// Leading byte is the number of "extra" bits at the end.
+			var bitCount int
+			var sawPipe bool
+			bits := []byte{0}
+			for i, r := range bitStr {
+				switch r {
+				case '0', '1':
+					if bitCount%8 == 0 {
+						bits = append(bits, 0)
+					}
+					if r == '1' {
+						bits[len(bits)-1] |= 1 << uint(bitCount%8)
+					}
+					bitCount++
+				case '|':
+					if sawPipe {
+						return token{}, &parseError{s.pos, errors.New("duplicate '")}
+					}
+
+					// Ensure that there aren't actually more than |padding| bits past the |.
```

Comment:
Nit: Looking through the rest of the project, I don't appear to have actually done the `|` thing in comments here.